basic-android-kotlin-compose-training-mars-photos-starter/app/src/main/java/com/example/marsphotos/ui/screens/realEstateScreen/components/RealEstateColumnScreen.kt

```diff
@@ -27,11 +31,13 @@ fun RealEstateColumnScreen(
                     navController = navController,
                     modifier = modifier
                         .padding(4.dp)
-                        .fillMaxWidth()
+                        .fillMaxWidth(),
```

Comment:
For RealEstateCard, why are there some parameters that are named (e.g. navController = navController, and some are just index instead of index = index) ? 

Any specific reason behind this approach?

This questions is for most of the custom compostables that you've made, not just this one.

Reply:
@cipry13 No reason in particular. I should have kept one approach or another, not to mix them together. I applied the change now.